core/src/main/java/discord4j/core/util/EntityUtil.java

```diff
@@ -69,10 +59,11 @@ public static ChannelBean getChannelBean(final ChannelResponse response) {
      */
     public static Channel getChannel(final ServiceMediator serviceMediator, final ChannelBean bean) {
```

Comment:
This is missing stores.

Reply:
Doesn't it need to handle every channel type to work properly? You added news.

Reply:
I'm sorry I misunderstood; I thought you meant stores as in storing channels. I realize now you meant the guild store channel type